projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/product-details-page.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as siteContextSelector from '../../helpers/site-context-selector';
+
+/**
+ * This test checks accessibility concerns on the PDP page using Access Continuum
+ */
+describe('Product Details Page Accessibility', { testIsolation: false }, () => {
+  beforeEach(() => {
```

Comment:
```suggestion
  before(() => {
```
Not navigating each time will make the test run a lot faster.

Reply:
Applied 👍